tests/test_zmanim.py

```diff
@@ -23,6 +26,8 @@ def test_bad_date(self):
 
     @pytest.mark.parametrize("execution_number", list(range(5)))
     def test_same_doy_is_equal(self, execution_number, random_date):
+        if _ASTRAL:
```

Comment:
Why skip test in this case?

Reply:
Done.

Note: I had to add a significant grace period, and below the reason for that.
Year is 365.25 days (which is the reason that there are leap years).
Astral takes that into account, so the same day in a different year is not identical in term of the times.
The main problem is that we calculate sun_hour as "(sunset - sunrise) // 12". It means that 59.9 will become 59, which can cause sun_hour of the same-day-different-years to have 1 minute delta (e.g. 59.9 vs 60.1). Now, we multiple this 1 minute difference in other places. For example, small_mincha=sunrise + 9.5 * sun_hour. It makes the delta quite significant.

Since we no longer do the actual arithmetic for time calculation, we can also consider the skip this test when Astral is used, as it tests that library (and not the logic in this library). Anyway, I kept it for now, and it's up to you to decide.